ui/raidboss/emulator/translations.ts

```diff
@@ -284,12 +339,14 @@ const emulatorLabels: Translation = {
     en: 'Hide Skipped',
     de: 'Verstecke Übersprungene Einträge',
     fr: 'Masquer les entrées sautées',
+    ja: 'スキップした項目を隠す',
     cn: '隐藏已跳过项目',
   },
   ' label[for=hideCollector]': {
     en: 'Hide Collectors',
     de: 'Verstecke "Sammel" Einträge',
     fr: 'Masquer les entrées "Collecté"',
+    ja: 'コレクターを隠す',
     cn: '隐藏收集器',
```

Comment:
@quisquous After talking with @ShadyWhite , we think we might translate wrong on this text, can you explain what is "collectors" means?

Reply:
Collectors are triggers that don't output anything.

Reply:
Oh, that's it. I'll open a new PR to fix it!